crates/ruma-client-api/src/media.rs

```diff
@@ -8,3 +8,38 @@ pub mod get_content_as_filename;
 pub mod get_content_thumbnail;
 pub mod get_media_config;
 pub mod get_media_preview;
+
+/// Checks whether a given `Content-Type` is considered "safe" for having a `Content-Disposition` of
+/// `inline` returned on `/download`.
```

Comment:
Could we add a link to the spec in these docs, to: https://spec.matrix.org/latest/client-server-api/#serving-inline-content (the link will not work until the next release, so we will have to wait to merge this)?